added dataset generation function and dataset

shubhamsrivast4u opened the pull request:
Dataset generated is in 'shubham/traffic_dataset'.
The function to generate dataset is in 'shubham/datasetgeneration.ipynb'. Change the 'num_samples' variable at the end of the code to get different dataset. If you select 'num_samples =100', then it will generate 50 samples of 7 consecutive frames in which collision has occcured in the 7th frame, and 50 samples of 7 consecutive frames with no collision.

shubhamsrivast4u committed:
shubham modified

shubhamsrivast4u committed:
added dataset

shubhamsrivast4u committed:
added dataset

shubhamsrivast4u committed:
Merge branch 'Nurassyl-lab:master' into master

Nurassyl-lab commented:
Github storage is limited. It is better to either not to add the dataset at all, and just provide a code to generate it. Or at least compress it into an archive. 1st one is preferable. You can use seeds to ensure that the dataset generate is reproducible.

Thanks!

shubhamsrivast4u committed:
Remove shubham/traffic_dataset from git tracking and add to .gitignore

shubhamsrivast4u committed:
Removed ipynb checkpoints from git tracking

shubhamsrivast4u commented:
Done all changes, removed traffic dataset folder. Added the code to generate dataset with constant speed vehicles and pedestrians in 'datasetgeneration-constantspeed.ipynb' file. Just run this file on your system. If you want, I can upload the dataset generated at my end on googledrive folder.

shubhamsrivast4u committed:
added yolo tracking with different cameras command and vehicle reidentification code
